docs/CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+  * **places**: Default `country` labels to **zoom 6** instead of 1 when there isn't Natural Earth match (Issue [#1826](https://github.com/tilezen/vector-datasource/issues/1826))
+  * **places**: Default `region` labels to **zoom 8** instead of 1 when there isn't Natural Earth match (Issue [#1826](https://github.com/tilezen/vector-datasource/issues/1826))
+
+  #### DOCUMENTATION CHANGES
+
+    TODO update JSON
+  * Updated Layers documentation for v1.7 schema changes.
+  * Publish docs to [tilezen.readthedocs.io](https://tilezen.readthedocs.io).
+  * Update documentation formatting for ReadTheDocs.io Markdown requirements.
+  * Restructure some file layout for ReadTheDocs.io build system, including: `CHANGELOG`, `LICENSE`, `CONTRIBUTING`, `MIGRATION_GUIDE`, `PERFORMANCE`, `SEMANTIC-VERSIONING`, and `TEST` files.
+  * Corrected order of `low_emission_zone` in list.
+  * Generalized `hgv_restriction` property units to mostly meters for heavy goods vehicle truck access restrictions.
+
+  #### INTERNAL CHANGES
+
+  * Add `gunicorn` to dependencies, thanks [@rwrx](https://github.com/rwrx). [PR #1690](https://github.com/tilezen/vector-datasource/pull/1690)
+  * Bump `PyYAML` version for CVE-2017-18342.
+  * Stop using `tags-> in YAML (Issue [#1199](https://github.com/tilezen/vector-datasource/issues/1199))
+  * Add ability to make tests from relations, and calculate the area for TODO
+  * Estimate `capacity` for parking lots and garages based on area and parking type with new `tz_estimate_parking_capacity` function.
+  * Protect against `None` shapes in way area calculation.
+  * Add `all_the_kinds` script to output all the `kind` and `kind_detail` values per `$layer`, with their `min_zoom` from the YAML files.
+  * Add whitelists to `kind_detail` in each layer YAML so that we can enumerate all possible values for the script.
+  * Extend Natural Earth test generator to support polygons.
+  * Use new `drop_names` post-process function to drop all the names, not just the TODO
+  * Add `CollisionRanker` to support YAML-based spreadsheets for `collision_rank` logic, including reserved, gaps, and filters across layers.
+  * Add `safe_int` implementation for SQL.
+  * Expose a test method that can return the whole tile. Used in new `collision_rank` tests.
+  * Clip to Mercator world bounds before projecting shapefiles.
+  * Deal with 'download only' test mode.
+  * Skip download only tests after downloading fixtures (if any).
+  TODO osm land/water
+  * Update assets bundle for #1552 & #1809, and updated OSMdata.com land, water.
+  * Reorder under specific national parks, but above the generic tier2 na…  …
+  * Guard against future airport runway polygons in landuse versus runway lines in roads `sort_rank` errors with a test.
+  * Don't create a zero area property on points in SQL templating.
+  * Support shapefiles in ZIP for asset bundle creation.
```

Comment:
Should be: Support for _multiple_ shapefiles in a single ZIP for asset bundle creation, link to #1809?

Reply:
Fixed in b80bf944028b77062e27a33f8114e43d2e0bc51d.